### ISSUE TITLE
[Localization] Missing Italian localization for quarkus.io

### ISSUE THREAD

**Issue by davidesalerno** [quarkusio/quarkusio.github.io#1918]
**Describe the bug**
At the moment quarkus.io is written in English and it is localized in these languages:

- [ja.quarkus.io](https://github.com/quarkusio/ja.quarkus.io) for Japanese
- [cn.quarkus.io](https://github.com/quarkusio/cn.quarkus.io) for Chinese (simplified)
- [es.quarkus.io](https://github.com/quarkusio/es.quarkus.io) for Spanish
- [pt.quarkus.io](https://github.com/quarkusio/pt.quarkus.io) for Brazilian Portuguese

An Italian version is missing.

**Comment by maxandersen** [comment]
all of these are done by volunteers. Are you up for starting the work for Italian ?

**Comment by davidesalerno** [comment]
@maxandersen yes, I'm available as a volunteer to work on the italian localization!

**Comment by maxandersen** [comment]
@davidesalerno - sorry - pto's got in our way :) can you join [zulip](https://quarkusio.zulipchat.com/) and ping me or join "translation/l10n" then can get it setup.
